lib/internal/http2/core.js

```diff
@@ -1055,9 +1063,9 @@ function createClientSession(options, socket) {
   socket.on('pause', socketOnPause);
   socket.on('drain', socketOnDrain);
 
-  session.on('error', sessionOnError);
-  session.on('stream', sessionOnStream);
-  session.on('selectPadding', sessionOnSelectPadding);
+  // session.on('error', sessionOnError);
+  // session.on('stream', sessionOnStream);
+  // session.on('selectPadding', sessionOnSelectPadding);
```

Comment:
@jasnell These events are certainly relevant to the client as a public interface. I used the `session:stream` event in one of the tests. But these internally assigned handlers were server-specific. It is not clear to me how this should be rewritten. Suggestion?